ui/src/components/settings/SettingRow.js

```diff
@@ -103,6 +103,9 @@ export default class SettingRow extends React.Component {
                         value={this.props.value}
                         checked={this.props.value}
                         onChange={this.toggleSwitchButton}
+                        disabled={
+                            this.props.disabled ? this.props.disabled : false
+                        }
```

Comment:
it look cleaner to write 
"this.props.disable || false"
has the same result

Reply:
done